lib/menu.rb

```diff
@@ -0,0 +1,32 @@
+require_relative 'dish'
+require 'csv'
+
+class Menu
+
+  attr_reader :dishes
+
+  def initialize
+    @dishes = []
+  end
+
+  def add_dish(dish)
+    @dishes << dish
+  end
+
+  def display
+    @dishes.each do |dish|
+    puts "#{dish.name}: £#{dish.price}"
+    end
+  end
+
+# Allows adding of menu items in bulk
+  def add_menu(filename = 'menu.csv')
```

Comment:
`menu.add_menu(filename)` doesn't seem right to me.  Maybe `menu.load(filename)` would be better.